### ISSUE TITLE
Is Eventlet Still The Best Option?

### ISSUE THREAD

**Issue by ebram96** [miguelgrinberg/Flask-SocketIO#2067]
I'm quoting from documentation:
"[eventlet](http://eventlet.net/) is the best performant option, with support for long-polling and WebSocket transports."

But when I check Eventlet's documetation I see:
"[New usages of eventlet are now heavily discouraged!](https://eventlet.readthedocs.io/en/latest/)"

I'm wondering if I'm starting to use Flask-SocketIO in a new project, should I still go with Eventlet?
If no, what is the recommendation here?

**Comment by miguelgrinberg** [comment]
No, eventlet is not a good option anymore, it is a project that is winding down and will eventually be closed. For a new project I would recommend you start with the threading mode and if that is insufficient you switch to gevent.
